apps/web/lang/de.json

```
@@ -0,0 +1,359 @@
+    "postalCodeLabel": "Postleitzahl",
+    "useAsShippingLabel": "Als Lieferadresse verwenden",
+    "selectPlaceholder": "-- Auswählen --",
+    "emailLabel": "E-Mail",
+    "passwordLabel": "Passwort",
+    "passwordHint": "8 oder mehr Zeichen mit einer Mischung aus Buchstaben, Zahlen und Symbolen.",
+    "termsAndConditionsLabel": "Hiermit bestätige ich, dass ich die {terms} gelesen habe.",
+    "subscriptionLabel": "ch möchte gelegentlich E-Mails über Sonderangebote, neue Produkte und exklusive Aktionen erhalten. Ich kann mein Abonnement jederzeit kündigen. (Optional)",
+    "asterixHint": "* markierte Zustimmungen sind erforderlich"
+  },
+  "myCart": "Mein Warenkorb",
+  "orderSummary": "Bestellübersicht",
+  "itemsInCart": "(Artikel: {count})",
+  "itemsSubtotal": "Zwischensumme der Artikel",
+  "estimatedTax": "MwSt.",
+  "apply": "Anwenden",
+  "clear": "Zurücksetzen",
+  "min": "Min",
+  "max": "Max",
+  "total": "Gesamt",
+  "goToCheckout": "Zur Kasse",
+  "delivery": "Versandkosten",
+  "emptyCart": "Ihr Warenkorb ist leer",
+  "emptyCartImgAlt": "Bild eines leeren Warenkorbs",
+  "backToShopping": "Zurück zum Einkaufen",
+  "allFromCategory": "Alle {name}",
+  "listSettings": "Listeneinstellungen",
+  "closeListSettings": "Listeneinstellungen schließen",
+  "clearFilters": "Alle Filter löschen",
+  "showProducts": "Produkte anzeigen",
+  "showMore": "Mehr anzeigen",
+  "showLess": "Weniger anzeigen",
+  "filter": "Filter",
+  "allProducts": "Alle Produkte",
+  "sortBy": "Sortieren nach",
+  "perPage": "Pro Seite",
+  "sortType": {
+    "nameA-Z": "Name A-Z",
+    "nameZ-A": "Name Z-A",
+    "priceUp": "Preis ⬆",
+    "priceDown": "Preis ⬇",
+    "reviewsUp": "Kundenbewertung ⬆",
+    "reviewsDown": "Kundenbewertung ⬇"
+  },
+  "numberOfProducts": "1 Produkt | {count} Produkte",
+  "prev": "Vorherige",
+  "next": "Nächste",
+  "emptyStateText": "Wir haben keine Produkte mehr in dieser Kategorie.",
+  "emptyStateText2": "Haben Sie nicht gefunden, was Sie suchen?",
+  "emptyStateAltText": "Bild das beschreibt, dass keine Produkte gefunden wurden",
+  "resultsFor": "Ergebnisse für \"{phrase}\"",
+  "prevAriaLabel": "Gehe zur vorherigen Seite",
+  "nextAriaLabel": "Gehe zur nächsten Seite",
+  "secondPageAriaLabel": "Zweite Seite",
+  "lastPageAriaLabel": "Letzte Seite",
+  "currentPage": "Seite {page} von {totalPages}",
+  "checkout": "Kasse",
+  "backToCart": "Zurück zum Warenkorb",
+  "back": "Zurück",
+  "buy": "Kaufen",
+  "termsInfo": "Es gelten unsere {terms}. Bitte nehmen Sie unser {cancellationRights} und {privacyPolicy} zur Kenntnis.",
+  "termsAndConditions": "AGB",
+  "privacyPolicy": "Datenschutzerklärung",
+  "cancellationRights": "Widerrufsrecht",
+  "contactInfo": {
+    "heading": "Kontaktinformationen",
+    "description": "Fügen Sie Ihre E-Mail-Adresse hinzu.",
+    "add": "Kontaktinformationen hinzufügen",
+    "email": "E-Mail",
+    "phone": "Telefonnummer",
+    "phonePlaceholder": "z.B. 123 456 7890",
```

```suggestion
    "phonePlaceholder": "z.B. 1234 567890",
```
I think the pattern for German numbers is different?